packages/jest-config/package.json

```diff
@@ -32,7 +32,7 @@
     "@babel/core": "^7.11.6",
     "@jest/test-sequencer": "workspace:^",
     "@jest/types": "workspace:^",
-    "@rwx-research/abq": "0.1.0-alpha.3",
+    "@rwx-research/abq": "0.1.0-alpha.5",
```

Comment:
This gets bumped further with @ayazhafiz's multiple-test-results-per-manifest-item changes.